styles/style.css

```diff
@@ -0,0 +1,133 @@
+header {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 2;
```

Comment:
Love to see it -- `z-index` is an incredibly useful tool (and the cause of many CSS bugs/fixes 😢 ) !